app/controllers/articles_controller.rb

```diff
@@ -23,10 +23,8 @@ def feed
       render body: nil
       return
     end
-
-    set_surrogate_key_header "feed"
-    response.headers["Surrogate-Control"] = "max-age=600, stale-while-revalidate=30, stale-if-error=86400"
-
+    set_surrogate_keys_by_case
+    response.headers["Surrogate-Control"] = "max-age=186400, stale-while-revalidate=3600, stale-if-error=86400"
```

Comment:
I haven't noticed so far but we're calling setting caching twice I think we should use declarative programming and keep using `:set_cache_control_headers` which does exactly that:

https://github.com/fastly/fastly-rails/blob/6ed13a694b8753d43515079ae667fe6f8c0fcd23/lib/fastly-rails/action_controller/cache_control_headers.rb#L10-L12

So, as the pattern around the code base is "set cache control headers in a before action and then set surrogate keys at the end of the controller", let's do it like this:

```ruby
before_action -> { set_cache_control_headers(186_400, stale_while_revalidate: 30, stale_if_error: 86_400) }, only: %i[feed]
```

or even better (personally I end up googling "186400 seconds in hours" everytime I see these numbers :D) we can do:

```ruby
  before_action only: %i[feed] do
    set_cache_control_headers(51.hours + 46.minutes + 40.seconds, stale_while_revalidate: 1.hour, stale_if_error: 24.hours)
  end
```


as we're already calling set cache control headers.

I think it's going to be clearer to the reader as how long we're caching our RSS feed for.

We might also want to add a test or not. 

BTW I've noticed, by testing this manually locally, that we're serving our feed with the service worker's shell. I've sent a PR to fix that: https://github.com/thepracticaldev/dev.to/pull/5734